editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid

```diff
@@ -18,7 +20,7 @@ There are several differences from the [[tm-new-tiddler message|WidgetMessage: t
 
 The ''action-createtiddler'' widget is invisible.
 
-<<.from-version "5.2.0">> The action widgets contained within the ''action-createtiddler'' widget are executed after the new tiddler has been created with the title of the tiddler in the variable `createTiddler-title` and `createTiddler-draftTitle`.
+<<.from-version "5.2.0">> The action widgets contained in the widget-body of the ''action-createtiddler'' widget are executed after the new tiddler has been created with the title of the tiddler in the variable `createTiddler-title` and `createTiddler-draftTitle`. See the last example.
```

Comment:
"widget-body" is not a term we use in the documentation. Other widgets use the phrase "...the content of the widget".